LPChoose.jl

```diff
@@ -1,12 +1,22 @@
 #include("LPChoose.jl"); LPChoose("smalldata.txt",100,0.00);
 using DataFrames,LinearAlgebra,CSV,DelimitedFiles,SparseArrays,GLPK,JuMP,Statistics, StatsBase
 using ProgressMeter
+
+import_Gurobi = false  # import Gurobi package when import_Gurobi = true
```

Comment:
We don't need import_Gurobi. I don't see you provide an interface for users for import_Gurobi. If it is available, we just load it.